docs/index.asciidoc

```diff
@@ -80,6 +80,12 @@ Any ongoing modification of the dictionary file should be done using a
 copy/edit/rename or create/rename mechanism to avoid the refresh code from
 processing half-baked dictionary content.
 
+==== Compatibility with the Elastic Common Schema (ECS)
+
+The plugin acts as an in place translator if `field` and `target` are the same
+and does not produce any new event fields.
```

Comment:
>The plugin acts as an in-place translator if `field` and `target` are the same and does not produce any new event fields.

This statement takes a minute to parse and could be clearer with a slight rewording:

I think this is what you mean: 
If `field` and `target` are the same, the plugin acts as an in-place translator  and does not produce any new event fields.